srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -0,0 +1,104 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.validation.Validator;
+
+import gov.va.med.srcalc.domain.model.RiskModel;
+import gov.va.med.srcalc.domain.model.Specialty;
+
+public class EditRiskModel implements Comparable<EditRiskModel>{
+
+	private final RiskModel riskModel;
+    private static final Logger fLogger = LoggerFactory.getLogger(EditRiskModel.class);
+
+    private String modelName;
+    // can't change the id so just reference the id from the RiskModel
+    //private final int id;
+    
+//    private RiskModel importedModel;
+    
+	protected EditRiskModel( final RiskModel rm ) 
+	{
+		riskModel = rm;
+		modelName = rm.getDisplayName();
+		//id = rm.getId();
+	}
+	
+	public static EditRiskModel fromRiskModel( final RiskModel rm ) 
+	{
+		return new EditRiskModel( rm );
+	}
+	
+	public RiskModel getRiskModel() 
+	{
+		return riskModel;
+	}
+	
+	public String getModelName( ) 
+	{
+		return modelName;
+	}
+
+	public void setModelName( String mn ) 
+	{
+		modelName = mn;
+	}
+
+	public String getId( ) 
+	{
+		return Integer.toString( riskModel.getId() );
+	}
+	
+	public List<Specialty> getSpecialties() 
+	{
+		return new ArrayList<Specialty>( );
```

Comment:
This is OK, but a general tip: `java.util.Collections` provides an optimized `emptyList()` method for this purpose.